frontend/src/pages/Graphing/GraphingEditor.tsx

```diff
@@ -712,6 +710,14 @@ export const GraphingEditor: React.FC = () => {
 		],
 	)
 
+	const setEditor = (e: Editor) => {
+		if (e === Editor.SqlEditor) {
+			setSqlInternal(convertSettingsToSql(settings, startDate, endDate))
+			setSql(convertSettingsToSql(settings, startDate, endDate))
```

Comment:
Ignore if I am missing something, but do we have to do this twice? Or can we do the conversion once and set to a variable?

```
const convertedSql = convertSettingsToSql(settings, startDate, endDate)
setSqlInternal(convertedSql)
setSql(convertedSql)
```

Reply:
nope I copy pasted this - will fix